README.md

```diff
@@ -1,2 +1,3 @@
 # platinum-elements
-Elements that provide features to turn your web page into a true webapp - with things like push notifications, offline usage, and more.
+
```

Comment:
It was intentional actually :hamster: 
It looks better in a text editor in my opinion.